src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs

```diff
@@ -272,12 +276,13 @@ public IEnumerable<Command> GetCommands()
 
         public void Init(string gameRoot)
         {
+            Thread.Sleep(10000);
```

Comment:
Does the server need this sleep when starting to sync with an other service?
Is it required to start the server?

Reply:
Not really, no, just forgotten debug measure. Removed